integration/bootstrap_test.go

```diff
@@ -0,0 +1,56 @@
+package integration
+
+import (
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	bootstrapBinaryName = "xds-relay"
+)
+
+func TestBootstrap(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "bootstrap integration tests suite")
+}
+
+var bootstrapTestCases = []TableEntry{
+	{
+		Description: "negative test: negative cache TTL",
+		Parameters: []interface{}{
+			"./integration/testdata/bootstrap_configuration_negative_cache_ttl.yaml",
+			"./integration/testdata/keyer_configuration_complete_tech_spec.yaml",
+			"debug",
+			true,
+			"invalid Cache.Ttl: value must be greater than or equal to 0s",
+		},
+	},
+}
+
+var _ = Describe("Integration tests for bootstrapping xds-relay", func() {
```

Comment:
since we won't have positive tests, we can simplify the tests to remove the `wantErr` param. 

I thought a bit about how we could have positive tests, but I couldn't find a simple enough way that doesn't involve waiting or being more chatty in the initialization of the server (which is brittle).